build.gradle

```diff
@@ -1180,6 +1180,16 @@ project('test:system') {
             systemProperty "imageVersion", "INVALID_IMAGE_VERSION"
         }
 
+        // validating that bookkeeper operator version >= 0.1.3
+        if (System.getProperty("bookkeeperOperatorVersion") ==~ /^(0)\.(1)\.(0|1|2)(?:-.+)/) {
+            systemProperty "bookkeeperOperatorVersion", "INVALID_BOOKKEEPER_OPERATOR_VERSION"
```

Comment:
Where is the INVALID_BK_OPERATION_VERSION variable used? Do we exit the gradle task if BK/Pravega Operator version is invalid?

Reply:
Please make sure to test these changes for both positive and negative usecases

Reply:
@pbelgundi both positive and negative usecases have been tested